src/main/kotlin/app/batch/StreamingWriter.kt

```diff
@@ -137,6 +167,15 @@ class StreamingWriter(private val cipherService: CipherService,
     @Value("\${manifest.output.directory:.}")
     private lateinit var manifestOutputDirectory: String
 
+    @Value("\${use.blocked.lz4:false}")
+    private lateinit var useBlockedLz4: String
+
+    @Value("\${use.framed.lz4:false}")
+    private lateinit var useFramedLz4: String
+
+    @Value("\${use.gzip:false}")
+    private lateinit var useGzip: String
+
```

Comment:
I feel like these should be switched via a profile - compressionLz4, compressionGzip, compressionBzip for example. Using booleans doesn't feel right as it's one or the other really.

Reply:
@DanielChicot think the above have been missed in terms of removing them..